### PR TITLE
Add Card Component

### PR DESCRIPTION
**Pull request by manojkumar8400**
Add Card Component.

1. Card With Badges
2. Card with dismiss
3. Cards with text overlay
4. Text only cards
5. Vertical Cards
6. Horizontal Cards
7. Cards with shadow

### CODE REVIEW

File: documentation/card/card.css [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+    .code-cntnr {
+        display: none;
+    }
+
+    .footer-btn {
+        font-size: 1.4rem;
+    }
+}
+
+#card-compo {
```
Comment (msdeshmukh009) [inline documentation/card/card.css:23]: Maybe stick to classes for styling.

**Review by msdeshmukh009 — APPROVED** [review]

**Review by Chakravarthi0 — APPROVED** [review]
Looks good to me

**Review by srishti-maurya — APPROVED** [review]
looks good

**Comment by KaustubhM1997** [comment]
Looks good!
Suggestion: the dismiss button might look better after removing the border.

**Comment by manojkumar8400** [comment]
Thank you guys to approved PR and giving your valuable suggestions.

**Pull request merged**